### PR TITLE
Update EIP-3779: Remaining changes moved to EIP-2315

### PR DESCRIPTION
**Pull request by gcolvin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-3779.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
-category: Core
 author: Greg Colvin (@gcolvin), Greg Colvin <greg@colvin.org>, Brooklyn Zelenka (@expede)
 discussions-to: https://ethereum-magicians.org/t/eip-3779-safe-control-flow-for-the-evm/6975
+status: Withdrawn
+withdrawal-reason: material moved to EIP-2315
+type: Standards Track
+category: Core
 created: 2021-08-30
-requires: 2315, 3540, 3670, 3779, 4200
+requires:
```
Comment (Pandapip1) [inline EIPS/eip-3779.md:12]:
```suggestion
requires: 2315, 3540, 3670, 3779, 4200
```

**Review by Pandapip1 — DISMISSED** [review]

File: EIPS/eip-3779.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ status: Withdrawn
 type: Standards Track
 category: Core
 created: 2021-08-30
+requires: 2315, 3540, 3670, 3779, 4200
```
Comment (Pandapip1) [inline EIPS/eip-3779.md:11]: CC @SamWilsn - `preamble-requires-status` shouldn't trigger when the status is withdrawn

**Comment by SamWilsn** [comment]
I thought `Withdrawn` EIPs were immutable?

**Comment by Pandapip1** [comment]
> I thought Withdrawn EIPs were immutable?

This EIP isn't withdrawn. It is being withdrawn.

**Comment by Pandapip1** [comment]
Huh... I guess it *was* already withdrawn?!? It wasn't when I made changes earlier (see https://github.com/ethereum/EIPs/pull/5381/commits/b61c388e7a13b073c2db0ab8237ae228c6ae7283)

**Comment by Pandapip1** [comment]
So I think what happened here is that two PRs were merged that made the same change, and when I updated the branch, suddenly the changes that appeared to be made in this PR disappeared due to them being in the base branch.

**Comment by Pandapip1** [comment]
Closing this since it appears to be a duplicate of some other PR.

**Pull request closed without merging**